lmnet/lmnet/networks/object_detection/lm_fyolo.py

```diff
@@ -167,74 +168,38 @@ def __init__(
             activation_quantize_kwargs(dict): Initialize kwargs for activation quantizer.
         """
 
-        super().__init__(
+        LMFYolo.__init__(
+            self,
             *args,
             **kwargs,
         )
 
-        self.quantize_first_convolution = quantize_first_convolution
-        self.quantize_last_convolution = quantize_last_convolution
-
-        activation_quantizer_kwargs = activation_quantizer_kwargs if not None else {}
-        weight_quantizer_kwargs = weight_quantizer_kwargs if not None else {}
-
+        BaseQuantize.__init__(
+            self,
+            activation_quantizer,
+            activation_quantizer_kwargs,
+            weight_quantizer,
+            weight_quantizer_kwargs,
+            quantize_first_convolution,
+            quantize_last_convolution,
+        )
         assert callable(weight_quantizer)
         assert callable(activation_quantizer)
 
-        self.weight_quantization = weight_quantizer(**weight_quantizer_kwargs)
-        self.activation = activation_quantizer(**activation_quantizer_kwargs)
-
         if self.quantize_last_convolution:
             self.before_last_activation = self.activation
         else:
             self.before_last_activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
 
-    @staticmethod
-    def _quantized_variable_getter(
-            weight_quantization,
-            quantize_first_convolution,
-            quantize_last_convolution,
-            getter,
-            name,
-            *args,
-            **kwargs):
-        """Get the quantized variables.
-
-        Use if to choose or skip the target should be quantized.
-
-        Args:
-            getter: Default from tensorflow.
-            name: Default from tensorflow.
-            weight_quantization: Callable object which quantize variable.
-            args: Args.
-            kwargs: Kwargs.
-        """
-        assert callable(weight_quantization)
-        var = getter(name, *args, **kwargs)
-        with tf.variable_scope(name):
-            if "kernel" == var.op.name.split("/")[-1]:
-
-                if not quantize_first_convolution:
-                    if var.op.name.startswith("block_1/"):
-                        return var
-
-                if not quantize_last_convolution:
-                    if var.op.name.startswith("block_last/"):
-                        return var
-
-                # Apply weight quantize to variable whose last word of name is "kernel".
-                quantized_kernel = weight_quantization(var)
-                tf.summary.histogram("quantized_kernel", quantized_kernel)
-                return quantized_kernel
-
-        return var
-
-    def base(self, images, is_training):
+#    def base(self, images, is_training):
```

Comment:
Why comment out ?
is it work ?

Reply:
I think that what the base function in `lm_fyolo` do is 
1) define the `custom_getter` function and 
2) call the superclass' base function

Because the arguments of base function (images and is_training) are not related with the `custom_getter` function at all, 
it would be better to define `custom_getter` only once in the initialization instead of define it every time base function is called
and call the superclass' base function directly.

Reply:
`LMFYoloQuantize.base()` do

1. define the `custom_getter` function 
2. set `tf.variable_scope` with `custom_getter`  
```
with tf.variable_scope("", custom_getter=custom_getter):
```
3. call the superclass' base function 

Need `2` to quantize network.


> Because the arguments of base function (images and is_training) are not related with the custom_getter function at all,
yes. 

> it would be better to define custom_getter only once in the initialization instead of define it every time base function is called
not: `base` function is also only once called. 
Why do you not define `custom_getter` in `BaseQuantize` ?

And, Why you leave comment out (not deletel) ?

Reply:
You're right.
Thank you for clarifying the second task. (set `tf.variable_scope` with `custom_getter`)
Unlike classification task, there is not a `base` class for the object detection task.
Accordingly, the base function in this class is necessary.
I'm gonna delete the comment out.

> Why do you not define `custom_getter` in `BaseQuantize` ?
> 

That's because the arguments used to create custom_getter depends on the classes.
e.g. some classes require `quantize_first_convolution`, some require `quantize_last_convolution` 
, while some return `tf.summary.histogram` of the quantized value.

> And, Why you leave comment out (not delete) ?

I'm also sorry for forgetting to delete it.
I planned to delete it after the Jenkins test pass.